db/migrate/20180331230509_add_loan_public_level_options.rb

```diff
@@ -0,0 +1,18 @@
+class AddLoanPublicLevelOptions < ActiveRecord::Migration[5.1]
+  def up
+    loan_public_level = OptionSet.find_or_create_by(division: Division.root, model_type: Loan.name,
+      model_attribute: 'public_level')
+    loan_public_level.options.destroy_all
+    loan_public_level.options.create(value: 'featured', position: 1,
+      label_translations: {en: 'Featured', es: 'Destacado'})
+    loan_public_level.options.create(value: 'hidden', position: 2,
+      label_translations: {en: 'Hidden', es: 'Oculto'})
+  end
+
+  def down
+    loan_public_level = OptionSet.find_or_create_by(division: Division.root, model_type: Loan.name,
+      model_attribute: 'public_level')
+    loan_public_level.options.destroy_all
+    loan_public_level.destroy
```

Comment:
Since the options already exist, this reverse operation doesn't actually undo the above migration, it would also need to recreate them with the old TODO values. If you redo the `up` migration so that it changes the values only (instead of destroying them first), you should be able to just have an empty `down` migration, since there's no situation where you would really want to roll back changing the translations to be more correct.

Reply:
Okay. I'll change it to an empty `down` function.